tests/performance/llama/llama.rb

```diff
@@ -31,7 +31,7 @@ def test_llama_inference
     deploy_app
     copy_query_file
 
-    warmup
+    # warmup
```

Comment:
No warmup anymore?

Reply:
Just removing temporarily, as the warmup also takes like 15 minutes 😂

Reply:
👍